src/Common/src/System/Net/Http/Http2/CopyToAspNetCore.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
```

Comment:
We generally assume bash in most of our scripts in this repo.

Reply:
I changed that based on https://github.com/dotnet/corefx/pull/42425#discussion_r343498168

Reply:
Bash is a POSIX shell complaint, so it is a superset of POSIX shell. `if [ .. ]` works in both, `if [[ .. ]]` works only in Bash. Unless there is a need need for `[[]]` (like pattern match or array access), `[]` works fine. Scripts in .NET core org use both `[]` and `[[]]` (latter one, when really needed).

Reply:
While true, I don't see the point here for this script, which will be used in a relatively advanced scenario unlike `build.sh`, etc. It's for inner-loop development of HTTP2, so this should just be fine, as it does it's job of syncing / mirroring between the two repos.